packages/frontend/src/context/MyContext.tsx

```diff
@@ -102,6 +119,47 @@ export const MyContextProvider = (
     address: address as `0x${string}`,
   });
 
+  const getStores = async () => {
+    const stores = new Map();
+
+    const logs = await publicClient.getLogs({
```

Comment:
So the default behavior of getLogs is to only get the Logs from the last block 
https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getlogs
```
fromBlock: QUANTITY|TAG - (optional, default: "latest") Integer block number, or "latest" for the last mined block, "safe" for the latest safe block, "finalized" for the latest finalized block, or "pending", "earliest" for not yet mined transactions.

toBlock: QUANTITY|TAG - (optional, default: "latest") Integer block number, or "latest" for the last mined block, "safe" for the latest safe block, "finalized" for the latest finalized block, or "pending", "earliest" for not yet mined transactions.
```
It look like it is assumed that this will return all the logs.